ThumbnailSequence.cpp

```diff
@@ -1560,7 +1560,7 @@ ThumbnailSequence::CompositeItem::CompositeItem(
 bool
 ThumbnailSequence::CompositeItem::incompleteThumbnail() const
 {
-	return dynamic_cast<IncompleteThumbnail*>(m_pThumb) != 0;
+	return m_pThumb == 0 && dynamic_cast<IncompleteThumbnail*>(m_pThumb) != 0;
```

Comment:
But using std::move(m_ptrDbg) only moves the error to the called constructor:
```
.....filters/page_split/Task.cpp:242:36: error: use of deleted function 'std::unique_ptr<_Tp, _Dp>::unique_ptr(const std::unique_ptr<_Tp, _Dp>&) [with _Tp = DebugImagesImpl; _Dp = std::default_delete<DebugImagesImpl>]'
```

Using -Wl,-export-dynamic as suggested doesn't change anything in scantailor and it really doesn't make sense having to rebuild qt with non-standard linker flags?